vm/systemSmartContracts/auction.go

```diff
@@ -683,6 +723,7 @@ func (s *stakingAuctionSC) stake(args *vmcommon.ContractCallInput) vmcommon.Retu
 	// do the optionals - rewardAddress and maxStakePerNode
 	if uint64(lenArgs) > maxNodesToRun*2+1 {
 		for i := maxNodesToRun*2 + 1; i < uint64(lenArgs); i++ {
+			//TODO: check if the args len is enough or a correct address must be provided
```

Comment:
yes, it is enough. Other components are checking if this address is of metachain - than rewards will go to protocol sustainability.

Reply:
ok